### DISCUSSION TITLE
autoscroll down

### DISCUSSION THREAD

**Discussion by chris-yooo** [neuefische/questions#57] [Web]
I have 1 2 problems with the autoscroll down, which works I would like him but still build in so that even with a new sent message he jumps this a small step down - another problem what falls into there is that if the useEffect runs through then he would also jump down to the latest message and thus prevents the reading when scrolling up. so must also be blocked a certain time ^^.

link to repo: https://github.com/chris-yooo/capstone-project

**Comment by CJAkkuec** [comment]
Hey there, I'm not entirely sure what's supposed to be autoscrolling, but I'm guessing there should be a chat window that's autoscrolling with each new message added beyond a certain overflow threshold? Are you sure the autoscrolling is necessary?

**Reply by chris-yooo** [reply]
yes definitely, if now 300 messages are inside and you have to scroll down first uff...

yes in the branch "autoscroll" i updated the last stand.

**Reply by chris-yooo** [reply]
until now it is so that the first time the page is loaded the messages come and he then scrolls down to the last.

then after sending a message should also be scrolled again - this does not work yet because I do not know exactly where this should be.

and then comes the useeffect the messages fetched - that would cause with a timer every 5 seconds again the scrolling to the bottom there must be a block pure :D
